openapi-contracts/src/test/resources/domains/conversation/v1/messages/ListMessageProductDto.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "a list message title value",
+  "sections": [
+    {
+      "title": "a list section title value",
+      "items": [
+        {
+          "item": {
```

Comment:
Just noting for reference as this question will be addressed with the doc team, but the `item` element should not exist

Reply:
Yes, question sent.
This is where dedicated payloads and unit tests will be helpful to ensure regression or not with OAS file lifecycle